template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/training/notebooks/TrainWithFeatureStore.py.tmpl

```diff
@@ -233,6 +264,30 @@ training_set = fs.create_training_set(
     exclude_columns=exclude_columns,
 )
 
+{{else}}
```

Comment:
Add hyphen

Reply:
see latest updates -- will be adding links to screenshots